docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1096 @@
+
+            /// Unset the callback
+            VoiceControlManagerClient.FeedbackStreaming -= feedbackStreaming;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set TTS streaming callback function to be called when the vc client sends audio streaming for TTS feedback:
+    ```csharp
+    /// callback
+    void vcTtsStreaming(object sender, VcTtsStreamingEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetVcTtsStreamingCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.VcTtsStreaming += vcTtsStreaming;
+
+            /// Unset the callback
+            VoiceControlManagerClient.VcTtsStreaming -= vcTtsStreaming;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+
+<a name="start_and_stop_recording"></a>
+## Start and stop recording
+
+You can start, stop, or cancel recording using voice control manager:
+
+-   To start recording, use the `Start()` method with `exclusiveCommandOption` as the parameter.
+    The connected voice control service starts recording and the voice control manager state is changed to `Recording`.
+    If the parameter `exclusiveCommandOption` value is true, voice control service recognizes in only exclusive commands.
+    This method must be called when the voice control manager is in the `Ready` state:
+
+    ```csharp
+    void StartManager(bool exclusiveCommandOption)
+    {
+        try
+        {
+            VoiceControlManagerClient.Start(exclusiveCommandOption);
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+-   To stop recording, use the `Stop()` method.
+    The recording stops and the voice control manager state is changed to `Processing`.
+    When the recognition command result is processed, the `RecognitionResult` event triggers, and the state changes back to `Ready`.
+    This method must be called when the voice control manager is in the `Recording` state:
+
+    ```csharp
+    void StopManager()
+    {
+        try
+        {
+            VoiceControlManagerClient.Stop();
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+-   To cancel recording, use the `Cancel()` method.
+    This method must be called when the voice control manager is in the `Recording` and `Processing` state:
+
+    ```csharp
+    void CancelManager()
+    {
+        try
+        {
+            VoiceControlManagerClient.Cancel();
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+
+<a name="send_requests"></a>
+## Send requests
+
+You can send requests using the voice control manager:
+
+-   To send the event information to the voice control engine in purpose of activating specific action, use the DoAction() method.
```

Comment:
Global comment:

@sungraejo Kindly use inline code block to highlight code elements in the sentence as:

Line 631:
... action, use the `DoAction()` method.

Line 649:
... use the `SendSpecificEngineRequest()` method.

Line 760:
... `VC_AUDIO_ID_BLUETOOTH` or `VC_AUDIO_ID_MSF` in string